<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="DisconnectButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Avant">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Avant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="PowerSlider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>110</number>
    </property>
    <property name="maximum">
     <number>239</number>
    </property>
    <property name="value">
     <number>160</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="barre_batterie">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>520</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QToolButton" name="Gauche">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="Droite">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Vitesse">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="Reculer">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse : </string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="video" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ConnectVideo</string>
    </property>
   </widget>
   <widget class="QToolButton" name="CamRight">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="CamLeft">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="CamUp">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::UpArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="CamDown">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>25</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::DownArrow</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QWebEngineView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
